### PR TITLE
added orient function for principal axes take 2

### PR DESCRIPTION
**Pull request by tonigi**
Now in util

### CODE REVIEW

File: htmd/molecule/util.py [hunk 1]
```diff
@@ -116,6 +116,42 @@ def molRMSD(mol, refmol, rmsdsel1, rmsdsel2):
...
+    ----------
+    mol :
+        The Molecule to be rotated
+    sel : 
+        Atom selection on which the rotation is computed
+
+    Examples
+    --------
+    >>> mol = tryp.copy()
+    >>> mol.orient("noh")
```
Comment (stefdoerr) [inline htmd/molecule/util.py]: you need to update the example

Comment (tonigi) [reply htmd/molecule/util.py]: done thanks 🦅

**Comment by tonigi** [comment]
By the way, this kind of fixes the unimplemented "rotate" in solvate()

**Comment by tonigi** [comment]
this pull request thing is maddening

**Pull request closed without merging**